### PR TITLE
Arizona Family Tax Credit

### LINKED ISSUE #2460: Arizona Credits
https://www.azleg.gov/ars/43/01073.htm

### PR DESCRIPTION
**Pull request by JudyZ13**
Fixes #2460 

### Summary
:memo::moneybag::eyeglasses:

This pull request fixes the Arizona credits and adds the corresponding parameters, variables, and tests. It introduces new parameter files for the amount and eligibility of the Arizona family income tax credit, which is a refundable credit for low-income residents. It also introduces new variable files for the components of the Arizona family income tax credit, which depend on the income, filing status, and number of dependents of the tax unit. Finally, it adds new test files for the Arizona credits, which contain unit tests with different input and output scenarios.

> _`Arizona` credit_
> _New parameters and tests_

### Walkthrough
*  Add changelog entry for fixing Arizona credits and bumping patch version ([link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
*  Implement Arizona family income tax credit, which is a refundable credit for low-income residents based on their income, filing status, and number of dependents ([link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-ff4afdf242485f1840244be383f7bafcb64ce8f02ed2388517f0c47b95269134L1-R9), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-6941dc0d4df26b58cd56d1aa8013b2d674fea5a5c08b973952f017b78b26d5dcL1-R30), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-8d2e103ba849ea6dde0fb16d39b2954f0990ccb647f9933d4c9feed0ceafa73aL1-R26), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-55fdcdfdde60ce67669a532bbffdcf51ff0e5eb456927a4e1ab4dfdb2508278dL1-R10), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-5f514ba8efe6942f0d4df65722f0a2582b4aa00a3cc24652fdc4b2babdb0d0f9L1-R10), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-2361656128e09b4782bd50e62d4ac6964c45bff609590a614c43e1eeb1960801R1-R26), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-85be45670ad3c3c08d775100a972e753e81846a2dae1e84bc7abd67cc7ec7829R1-R25), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-de61d1c352a4de39063afe40e72aae6a6924169a7420202f378d941d3eb9f840R1-R24), [link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-ebb28ba7fb578d8130d63fcb637409dd2ada43305521eca11e6ef87b4e39153aR1-R21))
*  Add unit tests for Arizona family income tax credit, covering different scenarios of income, filing status, and number of dependents ([link](https://github.com/PolicyEngine/policyengine-us/pull/2461/files?diff=unified&w=0#diff-193387efe5b26e3905a0ad2875b047bd0f8f7c7af2e81b034fd71fec5c6047bfL1-R93))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: Arizona Eligibility of Couple Filing Separately for Income Credits.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml:1]:
```suggestion
description: Arizona provides the family tax credit for separate filers with income below this threshold.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml:1]: Other yaml files similar description

File: policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+metadata:
+  period: year
+  unit: currency-USD
+  label: AZ household credit base for Couple Filing Separately
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml:5]:
```suggestion
  label: Arizona family tax credit separate maximum income
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/eligibility/separate.yaml:5]: similar in other yaml files

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/az/tax/income/credits/amount.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  period: year
+  label: Arizona Family Income Tax Credit amount
+  reference:
+    - title: Arizona State Legislature Title 43 - Taxation of Income, Chapter 10, Article 5, 43-1073. Family income tax credit (B)
+      href: https://www.azleg.gov/ars/43/01073.htm
+    - title: 2022 Arizona Tax Form Instructions Line50
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page22
+    #not available in tax forms
+values:
+    2022-01-01: 40
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/amount.yaml:13]: could you add 2021 values and the 2021 tax form reference as well

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+description: Arizona provides the following famiy income tax credit amoount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml:1]:
```suggestion
description: Arizona provides the following family income tax credit amount. 
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+description: Arizona provides the family tax credit for head of household filers with income below this threshold.
+metadata:
+  type: single_amount
+  threshold_unit: child
+  rate_unit: currency-USD
+  rate_period: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:6]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:6]: dont need the rate period in all yaml files

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
+description: Arizona provides the family tax credit for head of household filers with income below this threshold.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:1]:
```suggestion
description: Arizona qualifies head of household with income below this amount for the family tax credit.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:1]: all eligibility variables

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+description: Arizona provides the following maximum amount under the arizona family tax credit based on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:1]:
```suggestion
description: Arizona provides the following maximum amount under the arizona family tax credit, based on filing status.
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  2021-01-01: 240
+WIDOW:
+  2021-01-01: 240
+SEPARATE:
+  2021-01-01: 120
+reference:
+  - title: 43-1073. Family income tax credit (B)
+    href: https://www.azleg.gov/ars/43/01073.htm
+  - title: 2022 Arizona Tax Form Instructions Line50
+    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page22
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:22]:
```suggestion
    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page23
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:22]: please check page numbers in pdf files, the value of the specific parameter has to appear on that specific page

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  - title: 2021 Arizona Tax Form Instructions Line50
+    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2021_140i-2D.pdf#page21
+    #not available in tax forms
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:25]: delete everywhere, the values seem to appear

[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:25]: or am I am misreading?

File: policyengine_us/tests/policy/baseline/gov/states/az/tax/income/credits/az_family_tax_credit.yaml [hunk 10]
```diff
@@ -0,0 +1,59 @@
+- name: AZ credit unit test 1
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/az/tax/income/credits/az_family_tax_credit.yaml:1]: moving forward, the test cases should have a name that explains what edge case we are testing

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.credits.family_tax_credits
+        income = tax_unit("adjusted_gross_income", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:15]: we need to use arizona agi and not federal agi -> please double check in tax form and law

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_adjusted_gross_income.py [hunk 12]
```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class az_adjusted_gross_income(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_adjusted_gross_income.py:4]:
```suggestion
class az_agi(Variable):
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_adjusted_gross_income.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_adjusted_gross_income.py:1]: rename to az_agi.py

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml [hunk 13]
```diff
@@ -0,0 +1,34 @@
+description: Arizona qualifies head of household with income below this amount for the family tax credit.
+metadata:
+  type: single_amount
+  threshold_unit: child
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:4]:
```suggestion
  threshold_unit: dependent
```

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml [hunk 14]
```diff
@@ -0,0 +1,34 @@
+description: Arizona qualifies head of household with income below this amount for the family tax credit.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:1]: Also others
```suggestion
description: Arizona limits its family tax credit to head of household filers with income up to this amount, depending on the number of dependents.
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+metadata:
+  type: single_amount
+  threshold_unit: dependent
+  rate_unit: currency-USD
+  label: Arizona family tax credit head of household maximum income
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:6]:
```suggestion
  label: Arizona family tax credit head of household income limit
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml [hunk 16]
```diff
@@ -0,0 +1,24 @@
+description: Arizona provides the following maximum amount under the arizona family tax credit, based on filing status.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:1]:
```suggestion
description: Arizona limits its family tax credit to this total amount, depending on filing status.
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml:1]: Reorganize into `amount/{per_person,cap}.yaml`

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:1]: rename folder to `income_limit`

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+            [
+                p.eligibility.single,
+                p.eligibility.joint.calc(dependents),
+                p.eligibility.head_of_household.calc(dependents),
+                p.eligibility.separate,
+                p.eligibility.widow.calc(dependents),
+            ],
+        )
+        household_size = head + spouse + dependents
+        amount = p.amount * household_size
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:40]: can then drop lines 21, 22, and 39
```suggestion
        amount = p.amount * tax_unit("tax_unit_size", period)
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:1]: shard off an `az_family_tax_credit_eligible` variable for the income limit and use it in `defined_for`

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/eligibility/head_of_household.yaml:1]: duplicate file?

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount.yaml:1]: make a folder called `amount` and rename this file to `per_person.yaml`

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml:1]: rename this file to `cap` and include in amount folder

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/income_limit/separate.yaml [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+  label: Arizona family tax credit separate maximum income
+  reference:
+    - title: 43-1073. Family income tax credit (a) (9)
+      href: https://www.azleg.gov/ars/43/01073.htm
+    - title: 2022 Arizona Tax Form Instructions Line50
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page22
+    - title: 2021 Arizona Tax Form Instructions Line50
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2021_140i-2D.pdf#page21
+values:
+    2021-01-01: 10_000
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/income_limit/separate.yaml:14]:
```suggestion
  2021-01-01: 10_000
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py [hunk 19]
```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class az_family_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:10]: and remove from formula
```suggestion
    defined_for = "az_family_tax_credit_eligible"
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py [hunk 20]
```diff
@@ -0,0 +1,37 @@
...
+class az_family_tax_credit_eligible(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit Eligibility"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py:13]:
```suggestion
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/per_person.yaml [hunk 21]
```diff
@@ -0,0 +1,14 @@
...
+  label: Arizona Family Income Tax Credit amount
+  reference:
+    - title: Arizona State Legislature Title 43 - Taxation of Income, Chapter 10, Article 5, 43-1073. Family income tax credit (B)
+      href: https://www.azleg.gov/ars/43/01073.htm
+    - title: 2022 Arizona Tax Form Instructions Line50
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page23
+    - title: 2021 Arizona Tax Form Instructions Line50
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2021_140i-2D.pdf#page21
+values:
+    2021-01-01: 40
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/per_person.yaml:14]:
```suggestion
  2021-01-01: 40
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py [hunk 22]
```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class az_family_tax_credit_eligible(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit Eligibility"
+    unit = USD
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py:8]:
```suggestion
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py [hunk 23]
```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class az_family_tax_credit_eligible(Variable):
+    value_type = float
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py:5]:
```suggestion
    value_type = bool
```

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+    definition_period = YEAR
+    defined_for = "az_family_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.credits.family_tax_credits.amount
+        filing_status = tax_unit("filing_status", period)
+        status = filing_status.possible_values
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:18]:
```suggestion
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py [hunk 25]
```diff
@@ -0,0 +1,21 @@
...
+class az_family_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "az_family_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py:13]:
```suggestion
```

File: policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py [hunk 26]
```diff
@@ -0,0 +1,35 @@
...
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.credits.family_tax_credits
+        income = tax_unit("az_agi", period)
+        filing_status = tax_unit("filing_status", period)
+        status = filing_status.possible_values
+        dependents = tax_unit("tax_unit_dependents", period)
+        max_income = select(
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit_eligible.py:19]:
```suggestion
        income_limit = select(
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/per_person.yaml [hunk 27]
```diff
@@ -0,0 +1,14 @@
+description: Arizona provides the following family income tax credit amount.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/per_person.yaml:1]:
```suggestion
description: Arizona provides the following family income tax credit amount per person. 
```

File: policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/cap.yaml [hunk 28]
```diff
@@ -0,0 +1,24 @@
+description: Arizona limits its family tax credit to this total amount, depending on filing status.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/amount/cap.yaml:1]:
```suggestion
description: Arizona caps its family tax credit at this total amount, depending on filing status.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]
